### PR TITLE
DOCSP-34718: unpack arrays tutorial

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-34718>
Staging - https://preview-mongodbrustagir.gatsbyjs.io/node/DOCSP-34718-unpack-arrats/aggregation-tutorials/unpack-arrays/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/aggregation-tutorials/unpack-arrays.txt [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+In this tutorial, you can learn how to use the {+driver-short+} to
+construct an aggregation pipeline, perform the
+aggregation on a collection, and print the results by completing and
+running a sample app. This aggregation performs the following operations:
+
+- Unwind an array field into separate documents
+- Matches a subset of documents by a field value
+- Groups documents by common field values
+- Adds computed fields to each result document
```
Comment (rachel-mack) [inline source/aggregation-tutorials/unpack-arrays.txt:31]: Issue: Make these parallel

```suggestion
- Unwind an array field into separate documents
- Match a subset of documents by a field value
- Group documents by common field values
- Add computed fields to each result document
```

Comment (rachel-mack) [reply source/aggregation-tutorials/unpack-arrays.txt:31]: Suggestion: If there is reference documentation for these operations, link those pages here. I didn't see anything specific to this driver, but maybe in the server manual?

**Review by rachel-mack — APPROVED** [review]
Ran the tutorial and it worked as expected 👍 
A couple minor things:

File: source/aggregation-tutorials/unpack-arrays.txt (hunk 1 above)
Comment (rustagir) [reply source/aggregation-tutorials/unpack-arrays.txt:31]: The specific aggregation operators are linked in the tutorial steps, this section in the introduction is just a high level plan language explanation of the tutorial!

**Pull request merged**